docs/common-patterns.rst

```diff
@@ -157,14 +155,11 @@ to automatically go to the next stage when the
 function finishes.
 
 .. note::
-    **Modifier May be Skipped**.
-    Since modifiers are applied by simply replacing
-    code and not by using a function call,
-    the code in the transitionNext modifier
-    can be skipped if the function itself uses
-    return. If you want to do that, make sure
-    to call nextStage manually from those functions.
-
+    **Modifier May be Skipped (OLD)**
+    As of version 0.3.6, modifier code placed after a function (i.e. after "_") 
```

Comment:
Oh I am sorry, I think I was not clear enough: We only have this starting from version 0.4.0 (the next one).

Otherwise the PR is perfect!